server/src/test/java/org/elasticsearch/action/admin/indices/close/CloseIndexResponseTests.java

```diff
@@ -52,11 +66,28 @@ public void testBwcSerialization() throws Exception {
 
                 final AcknowledgedResponse deserializedResponse = new AcknowledgedResponse();
                 try (StreamInput in = out.bytes().streamInput()) {
+                    in.setVersion(out.getVersion());
                     deserializedResponse.readFrom(in);
                 }
                 assertThat(deserializedResponse.isAcknowledged(), equalTo(response.isAcknowledged()));
             }
         }
+        {
+            final CloseIndexResponse response = randomResponse();
+            try (BytesStreamOutput out = new BytesStreamOutput()) {
+                out.setVersion(Version.V_7_1_0);
```

Comment:
Should this by 7.2 now? Or should the two assertions below (shards acknowledged and indices empty) simply be added to above block instead?

I guess this will change once backported to 7.x so OK to leave for now if that is the intention?

Reply:
fixed in https://github.com/elastic/elasticsearch/pull/39687/commits/de4e32db5036687a104bf7894c82fa78919d972a